### DISCUSSION TITLE
Miltiple DIscounts for the cart

### DISCUSSION THREAD

**Discussion by maxpanakov** [medusajs/medusa#3064] [Feature Requests]
Hey, is it possible to have multiple discounts for the cart ? I know that we can have them only if they have a different type(lshiping and total as an example) but we can't have multiple discount for total. Example: if customer registed he has a 5% discount, but he also can applay fixed 100usd discount. Or when we have a bundle discount for items  and also fixed discount for a total.

**Comment by maxpanakov** [comment]
Rn we just merge two discount in to one

**Comment by olivermrbl** [comment]
We currently do not support multiple discounts on a single cart - aside from the combination of a standard one + free shipping. We plan to revisit our promotions engine, but unfortunately, not in the foreseeable future.
